src/12.datetimeformat.js

```diff
@@ -854,7 +854,14 @@ function CreateDateTimeParts(dateTimeFormat, x) {
                     fv = ''; // TODO
                     break;
 
-                    // TODO: Era
+                  case 'era':
+                    try {
+                      fv = resolveDateString(localeData, ca, 'eras', f, tm['[['+ p +']]']);
```

Comment:
interesting, the spec doesn't list `"era"` in the list of things to keep in mind: https://tc39.github.io/ecma402/#sec-datetimeformat-abstracts

I wonder if we need to revisit the spec in that regards.